.codacy.yml

```diff
@@ -3,10 +3,5 @@ exclude_paths:
   - 'doc/joss-paper/**'
   - 'etc/**'
   - 'lib/cylc/tests/**'
-  - 'lib/isodatetime/**'
-  - 'lib/jinja2/**'
-  - 'lib/markupsafe/**'
-  - 'lib/parsec/tests/**'
-  - 'licences/**'
```

Comment:
This was added for parsec, but then I realized we were missing a few more lines from other folders removed recently.